aws_google_auth/__init__.py

```diff
@@ -28,6 +28,7 @@ def parse_args(args):
     parser.add_argument('-R', '--region', help='AWS region endpoint ($AWS_DEFAULT_REGION)')
     parser.add_argument('-d', '--duration', type=int, help='Credential duration ($DURATION)')
     parser.add_argument('-p', '--profile', help='AWS profile (defaults to value of $AWS_PROFILE, then falls back to \'sts\')')
+    parser.add_argument('-P', '--print_creds', dest="print_creds", action='store_true', default=False, help='Print the credentials.')
```

Comment:
Make the flag `print-creds` not with the underscore `_` and remove `dest`  - args parser will handle the python name convesion.